Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -124,10 +124,11 @@ const EquipmentBigFixInfo = (props: IProps) => {
       return;
     }
 
-    const sortedResult = Object.keys(response)
+    const sortedResult = Object.keys(response.result[0])
       .sort()
       .reduce((accumulator, currentValue) => {
-        accumulator[currentValue] = response[currentValue];
+        const key = currentValue.replace('hardware_', '');
+        accumulator[key] = response.result[0][currentValue];
```

Comment:
I think you were going to change this to results (plural) since it is a list.  But otherwise this PR looks good.